tests/integration_tests/test_cli.py

```diff
@@ -106,16 +110,51 @@ def test_check_configs(self):
             cli.check_config(config)
 
     def test_overview( self ):
-        bittensor.subtensor.register = MagicMock(return_value = True)  
-        
-        config = self.config
-        config.command = "overview"
-        config.subtensor._mock = True
-        config.subtensor.network = "mock"
-        config.no_prompt = True
+        # Mock IO for wallet
+        with patch('bittensor.Wallet.coldkeypub_file', MagicMock(
+            exists_on_device=MagicMock(
+                return_value=True # Wallet exists
+            )
+        )):
+            bittensor.subtensor.register = MagicMock(return_value = True)  
+            
+            config = self.config
+            config.wallet.path = '/tmp/walletpath'
```

Comment:
see my other comment re: wallet path and parallel tests. Might not do anything right away, but could cause intermittent test failures down the road.